src/polyswarm/scan.py

```diff
@@ -61,9 +65,11 @@ def file(ctx, recursive, timeout, nowait, path):
               help='How long to wait for results (default: {}).'.format(const.DEFAULT_SCAN_TIMEOUT))
 @click.option('-n', '--nowait', is_flag=True,
               help='Does not wait for the scan window to close, just create it and return right away.')
+@click.option('-s', '--scan-config', type=click.STRING, default=None,
```

Comment:
I think the default for URL should be something that makes it work, rather than something that will fail half the time.